configuration/framework_config.cfg

```diff
@@ -45,11 +45,11 @@ TARGETS_DIR: targets
 
 
 # ---------------------- Default profile settings ---------------------- #
-DEFAULT_GENERAL_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/general/default.cfg
-DEFAULT_RESOURCES_PROFILE: @@@FRAMEWORK_DIR@@@/profiles/resources/default.cfg
+DEFAULT_GENERAL_PROFILE: ~/.owtf/configuration/general.cfg
+DEFAULT_RESOURCES_PROFILE: ~/.owtf/configuration/resources.cfg
+DEFAULT_MAPPING_PROFILE: ~/.owtf/configuration/mappings.cfg
```

Comment:
Silly question but in the case we want to support Windows, wouldn't `~/` fail? Can't we use the `os` module to expand the user home directory for us?